app/demo-geant-integration/SensitiveDetector.cc

```diff
@@ -41,9 +54,29 @@ G4bool SensitiveDetector::ProcessHits(G4Step* step, G4TouchableHistory*)
         return false;
     }
 
-    auto time = step->GetPreStepPoint()->GetGlobalTime();
+    // Fill the hit data for this step
+    auto         touchable = step->GetPreStepPoint()->GetTouchable();
```

Comment:
If you're expecting an `auto` type to be a pointer, it's more readable to write it as:
```suggestion
    auto* touchable = step->GetPreStepPoint()->GetTouchable();
    CELER_ASSERT(touchable);
```
(or if it's a reference to a smart pointer, `auto&`, or `auto&&` if you might be getting it by value and might get by reference)

Definitely also add an assertion here, because if I mess up the SD callback code, the touchable handle will be a null pointer 😬

Reply:
Done as suggested!